opossum-tool/OpossumUI
# feat: make project statistics popup optional on startup

### Summary of changes
- introduce electron app user settings
- allow user to toggle preference of showing project statistic on startup
- create state and hook for using a redux-persisted variable
- create hook for working with user settings
- create CLI arg to start app without project statistics and use it to simplify E2E tests

![image](https://github.com/opossum-tool/OpossumUI/assets/46091730/c723373c-6354-4caf-bd2c-a30c7af58ce4)

### Context and reason for change
Closes #2351 .

### How can the changes be tested
Change the new checkbox inside the project statistics popup and observe the effect when opening a file.
